src/uct/cuda/cuda_ipc/cuda_ipc_md.c

```diff
@@ -174,6 +175,19 @@ static ucs_status_t uct_cuda_ipc_rkey_release(uct_component_t *component,
                                               uct_rkey_t rkey, void *handle)
 {
     ucs_assert(NULL == handle);
+
+    uct_cuda_ipc_key_t *key = (uct_cuda_ipc_key_t *) rkey;
+    if (key->unmap_memhandle_func != 0 && (void *)key->d_mapped != 0) {
```

Comment:
please use `()` for each statement

Reply:
and check for `NULL`:
```
if ((key->unmap_memhandle_func != NULL) && ((void*)key->d_mapped != NULL)) {
```
or better (remove `(void*)`)
```
if ((key->unmap_memhandle_func != NULL) && (key->d_mapped != 0)) {
```